rolling/distribution.yaml

```diff
@@ -1230,6 +1230,21 @@ repositories:
       url: https://github.com/swri-robotics/marti_messages.git
       version: dashing-devel
     status: developed
+  mavlink:
+    doc:
+      type: git
+      url: https://github.com/mavlink/mavlink-gbp-release.git
+      version: release/rolling/mavlink
```

Comment:
Same comments as https://github.com/ros/rosdistro/pull/29422#pullrequestreview-652457659

Reply:
Based on the exchange there, I'll go ahead and merge this.